.github/workflows/dev-new.yaml

```
@@ -0,0 +1,55 @@
+
+      - name: Login to DockerHub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+      - name: Load .env from GitHub Secret
+        run: echo "${{ secrets.MY_ENV_VARS }}" > .env
+
+      - name: Build and push
+        uses: docker/build-push-action@v2
```

actionlint (1.7.4)

the runner of "docker/build-push-action@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)